src/test/kotlin/org/rust/ide/annotator/RsEdition2018KeywordsAnnotatorTest.kt

```diff
@@ -27,7 +28,6 @@ class RsEdition2018KeywordsAnnotatorTest : RsAnnotatorTestBase(RsEdition2018Keyw
         }
     """)
 
-    @MockEdition(CargoWorkspace.Edition.EDITION_2018)
```

Comment:
Maybe we should have something like `MinEdition` instead not to run the particular test if edition is lower than declared in annotation? Note, we already have similar `MinRustcVersion`
Currently, I can pass `DEFAULT_EDITION_FOR_TESTS=2015` that breaks this test

Reply:
I don't think we should introduce `MInEdition` for supporting `DEFAULT_EDITION_FOR_TESTS=2015` because in order to do that we should mark all or almost all tests currently marked with `@MockEdition(EDITION_2018)` as `@MinEdition(EDITION_2018)` and each new test also should be marked with `MinEdition` if it uses 2018 module paths. So I would support only 2018/2021 for `DEFAULT_EDITION_FOR_TESTS` since we don't plan to use `DEFAULT_EDITION_FOR_TESTS=2015` on CI